docs/openstack/nova_adoption.md

```diff
@@ -0,0 +1,143 @@
+          customServiceConfig: |
+            [workarounds]
+            disable_compute_service_check_for_ffu=true
+        schedulerServiceTemplate:
+          customServiceConfig: |
+            [workarounds]
+            disable_compute_service_check_for_ffu=true
+        cellTemplates:
+          cell0:
+            conductorServiceTemplate:
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+          cell1:
+            metadataServiceTemplate:
+              enabled: false # enable here to run it in a cell instead
+              override:
+                  service:
+                    metadata:
+                      annotations:
+                        metallb.universe.tf/address-pool: internalapi
+                        metallb.universe.tf/allow-shared-ip: internalapi
+                        metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+                    spec:
+                      type: LoadBalancer
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+            conductorServiceTemplate:
+              customServiceConfig: |
+                [workarounds]
+                disable_compute_service_check_for_ffu=true
+  '
+  ```
+
+* Wait for Nova control plane services' CRs to become ready:
+
+  ```bash
+  oc get nova --field-selector metadata.name=nova-api -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaapis --field-selector metadata.name=nova-api -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novacells --field-selector metadata.name=nova-cell0 -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novacells --field-selector metadata.name=nova-cell1 -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaconductors --field-selector metadata.name=nova-cell0-conductor -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaconductors --field-selector metadata.name=nova-cell1-conductor -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novametadata --field-selector metadata.name=nova-metadata -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novanovncproxies --field-selector metadata.name=nova-cell1-novncproxy -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaschedulers --field-selector metadata.name=nova-scheduler -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
```

Comment:
I would simplify this to:
```
oc wait --for condition=Ready --timeout=300s Nova/nova
```
If it is not enough to watch for the Ready condition of the Nova CR then that is a bug and please file it to nova-operator.

Reply:
done